axon-server/administration/multi-context.md

```diff
@@ -4,7 +4,56 @@
+
+| property | default value | description |
+| :---- | :---- | :---- |
+| event.storage |  [axoniq.axonserver.event.storage]/[context-name] | Location where the event store's event information will be stored |
+| event.segment-size | 250MB | Size of the event store's event segments |
+| event.index-format | JUMP_SKIP_INDEX | Index type used for the events this context (JUMP_SKIP_INDEX or BLOOM_FILTER_INDEX |
+| event.max-bloom-filters-in-memory | 100 | Number of bloom filters for events to be kept in memory (only applicable if index is BLOOM_FILTER_INDEX) ||
+| event.max-indexes-in-memory | 50 | Number of index files for events to be kept in memory | 
+| event.retention-time | 7d | Retention time for the events in the primary locations (if secondary location specified) |
+| snapshot.storage |  [axoniq.axonserver.snapshot.storage]/[context-name] | Location where the event store's event information will be stored |
+| snapshot.segment-size | 250MB | Size of the event store's snapshot segments |
+| snapshot.index-format | JUMP_SKIP_INDEX | Index type used for the snapshots this context (JUMP_SKIP_INDEX or BLOOM_FILTER_INDEX |
+| snapshot.max-bloom-filters-in-memory | 100 | Number of bloom filters for snapshots to be kept in memory (only applicable if index is BLOOM_FILTER_INDEX) ||
+| snapshot.max-indexes-in-memory | 50 | Number of index files for snapshots to be kept in memory | 
+| snapshot.retention-time | 7d | Retention time for the snapshots in the primary locations (if secondary location specified) |
+
+## Index formats
+
+As of version 4.4, Axon Server has a new format for the index of events and snapshots, called JUMP_SKIP_INDEX. This is the default format
+for all contexts that are created from this version. This index format uses a global index to locate the last event for a specific aggregate, 
+and maintains per segment per aggregate the location of the previous event. It improves the efficiency in looking up aggregates that are 
+distributed over segments that are further apart. For instance if you have 2000 event segments and an aggregate has events in segment 1500, 1000 and 500,
+using this index, Axon Server will find the latest event using the global index, and then from the index for segment 1500 that the previous event is in
+segment 1000. This prevents checking (the indexes of) all the files in between. When using this index, Axon Server will no longer create bloom filter files.
+For existing contexts the index format will remain BLOOM_FILTER_INDEX.
+ 
+>  _Note that when you delete an existing context with the preserve data option and then recreate it, without specifying the index format, 
+ Axon Server will use the JUMP_SKIP_INDEX format. This means that it will create a new index for the existing data, if the old format was BLOOM_FILTER_INDEX.
+ Depending on the size of the event store this can take a long time_
```

Comment:
```suggestion
> **Pre-4.4 Context Deletion**
> 
> Note that when you delete an existing context with the preserve data option and then recreate it, without specifying the index format, 
> Axon Server will use the JUMP_SKIP_INDEX format. This means that it will create a new index for the existing data, if the old format was BLOOM_FILTER_INDEX.
> Depending on the size of the event store this can take a long time.
```